conference.js

```diff
@@ -2606,7 +2617,10 @@ export default {
             email: formattedEmail
         }));
 
-        APP.settings.setEmail(formattedEmail);
+        APP.store.dispatch(updateSettings({
```

Comment:
I'd like to see this whole `changeLocalEmail` disappear. I think the profile side panel could dispatch an action to update it directly, instead of going through APP. Same goes for display name.